scripts/build-glean-metadata.py

```diff
@@ -10,8 +11,12 @@
 REPO_URL = PROBE_INFO_BASE_URL + "/glean/repositories"
 PINGS_URL_TEMPLATE = PROBE_INFO_BASE_URL + "/glean/{}/pings"
 METRICS_URL_TEMPLATE = PROBE_INFO_BASE_URL + "/glean/{}/metrics"
+DEPENDENCIES_URL_TEMPLATE = PROBE_INFO_BASE_URL + "/glean/{}/dependencies"
+
 OUTPUT_DIRECTORY = os.path.join("public", "data")
 
+default_dependencies = ['glean']
```

Comment:
Let's uppercase this to make sure it's part of the default definitions.

```suggestion
DEFAULT_DEPENDENCIES = ['glean']
```